frontend/src/scenes/plugins/edit/PluginDrawer.tsx

```diff
@@ -177,6 +177,17 @@ export function PluginDrawer(): JSX.Element {
                                                 <UploadField />
                                             ) : fieldConfig.type === 'string' ? (
                                                 <Input />
+                                            ) : fieldConfig.type === 'selection' ? (
```

Comment:
Commenting here out of the 3 [possible](https://github.com/PostHog/plugin-scaffold/pull/7/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR72) [places](https://github.com/PostHog/taxonomy-plugin/pull/1/files#diff-7b32f67ab72ef7b295c60be75ff703b7660e0f7e737e77251e5cc69deb7844aaR11-R18). 

Since it's the last chance to make a change in naming, what about instead of `type="selection"` and `options: []` using `type="choice"` and `choices: []` like so:

```js
[
    // with selection
    {
        "key": "conversionType",
        "name": "Conversion Type",
        "type": "selection",
        "options": ["camelCase", "PascalCase", "bookcase"],
        "hint": "What format the converted key will take"
    },
    // with choice
    {
        "key": "conversionType",
        "name": "Conversion Type",
        "type": "choice",
        "choices": ["camelCase", "PascalCase", "bookcase"],
        "hint": "What format the converted key will take"
    },

    // other possibilities
    {
        "key": "maxmindMmdb",
        "markdown": "Sign up for a [MaxMind.com](https://www.maxmind.com) account, download and extract the database and then upload the extracted `.mmdb` file below.\n\nYou can either download the large city databases (GeoIP2 City or GeoLite2 City) or the smaller country databases (e.g. GeoLite2 Country).\n\nSome PostHog installations might have problems loading the huge (70+MB) city databases, so unless you really need the additional data, we recommend installing the smaller (3MB) country database instead.",
        "name": "GeoIP .mddb database",
        "type": "attachment",
        "hint": "The \"GeoIP2 City\", \"GeoLite2 City\", \"GeoIP2 Country\", \"GeoLite2 Country\" .mmdb database file",
        "required": true
    },
    {
        "key": "localhostIP",
        "name": "IP to use instead of 127.0.0.1",
        "type": "string",
        "hint": "Useful if testing locally",
        "default": "",
        "required": false
    },
]
```

Otherwise, this looks good to me. We'll soon need to adapt it to support titles for the options, but that's for another day.

Reply:
I do like `choices` as well, happy to change.

As for the options, there's definitely a few things I could see being useful, such as having option be like:

{
    name: "Appears on select box",
    value: "The actual value passed to the config",
    description: "Appears in a tooltip"
}

Reply:
Updated to choice!